helm/0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+# Helm Job-Executor Integration
+
+This integration shows you how to leverage [Job-executor-service](https://github.com/keptn-contrib/job-executor-service) for deployment with `helm` on your Kubernetes cluster.
+
+**Please note, that the instructions provided here serve the purpose of enabling you to run and configure your Kubernetes deployments using job-executor-service. They are in no way intended to be a complete reference of Kubernetes, nor Helm, nor Keptn.**
+
+## Installation and Configuration
+
+### Step 1: Install the Job-Executor in your cluster
+
+Install [Job-Executor documentation](https://github.com/keptn-contrib/job-executor-service), and make sure it is subscribed to the following Keptn Cloud Events:
```

Comment:
Do you really mean "Install the ... documentation"?

Reply:
fixed